### PR TITLE
Parts ut

### PR DESCRIPTION
**Pull request by lewisrenfrew**
- nowhere near finished but PRing now since getting big

### CODE REVIEW

File: src/Domain.LinnApps/Parts/Part.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
     public class Part
     {
-        public int BridgeId { get; set; }
+        public long Id { get; set; }
```
Comment (richardiphillips) [inline src/Domain.LinnApps/Parts/Part.cs]: won't int be enough? I don't think we'll have a billion parts too soon

File: src/Domain.LinnApps/ParetoClass.cs [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
+
+    using Domain.LinnApps.Parts;
+
     public class ParetoClass
     {
         public string ParetoCode { get; set; }
 
         public string Description { get; set; }
+
+        public List<Part> Parts { get; set; }
```
Comment (richardiphillips) [inline src/Domain.LinnApps/ParetoClass.cs]: does this have to be list.

File: src/Resources/PartResource.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    {
+        public long Id { get; set; }
+
+        public string PartNumber { get; set; }
+
+        public string Description { get; set; }
+
+        public string RootProduct { get; set; }
+
+        public string StockControlled { get; set; }
```
Comment (richardiphillips) [inline src/Resources/PartResource.cs]: may be better as bools?

File: src/Domain.LinnApps/Parts/Part.cs (hunk 1 above)
Comment (lewisrenfrew) [reply src/Domain.LinnApps/Parts/Part.cs]: int is definitely enough! I was playing around with hashing part numbers to make aId's (which ended up being very big numbers) and forgot to revert back to int. Will do now.

**Pull request merged**